client/components/mobile/ActionStrip.jsx

```diff
@@ -25,42 +21,28 @@ const BottomBarContent = styled.div`
   }
 `;
 
-// Maybe this component shouldn't be connected, and instead just receive the `actions` prop
-const ActionStrip = ({ toggleExplorer }) => {
-  const { expandConsole, collapseConsole } = bindActionCreators(IDEActions, useDispatch());
-  const { consoleIsExpanded } = useSelector(state => state.ide);
-
-  const actions = [
-    {
-      icon: TerminalIcon, inverted: true, aria: 'Open terminal console', action: consoleIsExpanded ? collapseConsole : expandConsole
-    },
-    { icon: FolderIcon, aria: 'Open files explorer', action: toggleExplorer }
-  ];
-
-  return (
-    <BottomBarContent>
-      {actions.map(({
-        icon, aria, action, inverted
-      }) =>
-        (
-          <IconButton
-            inverted={inverted}
-            className={inverted && 'inverted'}
-            icon={icon}
-            aria-label={aria}
-            key={`bottom-bar-${aria}`}
-            onClick={() => action()}
-          />))}
-    </BottomBarContent>
-  );
-};
+const ActionStrip = ({ actions }) => (
+  <BottomBarContent>
+    {actions.map(({
+      icon, aria, action, inverted
+    }) =>
+      (<IconButton
+        inverted={inverted}
+        className={inverted && 'inverted'}
+        icon={icon}
+        aria-label={aria}
+        key={`bottom-bar-${aria}`}
+        onClick={() => action()}
```

Comment:
you don't need to create an anonymous function here, it can just be `onClick={action}`